adapter-guide/develop-transmission-module.md

```diff
@@ -0,0 +1,160 @@
+# STIX Transmission
+
+The steps below assume you have renamed the `async_dummy` module directory to our example connector name, `abc_security_monitor`.
+
+{: #transmission-mod}
+
+1. [Exploring the stix_transmission directory](#step-1-exploring-the-stix_transmission-directory)
+1. [Edit the apiclient.py file](#step-2-edit-the-apiclient-file)
+1. [Edit the dummy_connector.py file](#step-3-edit-the-dummy-connector-file)
+1. [Edit the dummy_error_mapper.py file](#step-4-edit-the-dummy-error-mapper-file)
+1. [Verify the implementation of each transmission method](#step-5-verify-each-transmission-method)
+
+## Step 1. Exploring the stix_transmission directory
+
+Verify that your `stix_transmission` directory contains the following folders and files.
+
+For an asynchronous transmission module, you must have the following files:
+
+| Folder/file        | Why is it important? Where is it used?                                  |
+| -------------------| ----------------------------------------------------------------------- |
+| \_\_init\_\_.py    | This file is required by Python to properly handle library directories. |
+| api_client.py       | Contains methods that make the data source API calls, used by the individual connector classes
+| query_connector.py | Contains class for executing a search on the data source
+| status_connector.py | Contains class for checking the status an active search on the data source
+| delete_connector.py | Contains class for deleting an active search on the data source
+| results_connector.py | Contains class for fetching the search results from the data source
+| ping_connector.py | Contains class to ping the data source
+| error_mapper.py |
+
```

Comment:
for sync connector that may be mentioned: https://github.com/opencybersecurityalliance/stix-shifter/blob/master/stix_shifter_modules/synchronous_dummy/entry_point.py#L20-L28
- self.set_async(False)
- results_connector = SynchronousDummyResultsConnector(api_client)
- delete_connector = SynchronousDummyDeleteConnector(api_client)